### PR TITLE
Add recipe for featuretools-tsfresh-primitives

### PR DESCRIPTION
**Pull request by tamargrey**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/featuretools-tsfresh-primitives/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - python >=3.7
+    - tsfresh >=0.15.0
+    - pandas >=1.3.0,<2.0.0
+    - featuretools >=1.0.0 
+    - statsmodels <=0.12.2 
+
+test:
+  imports:
+    - featuretools_tsfresh_primitives
+  requires:
```
Comment (synapticarbors) [inline recipes/featuretools-tsfresh-primitives/meta.yaml:31]: Are the packages outside of `pytest` needed to actually run the tests? I see that they are in the `test_requires`, but they don't look like you actually need them to run the `pytest` command.

Comment (tamargrey) [reply recipes/featuretools-tsfresh-primitives/meta.yaml:31]: Nope--removed! Thanks for pointing out.

**Comment by thehomebrewnerd** [comment]
I am willing to be a maintainer.

**Comment by gsheni** [comment]
I am willing to be a maintainer.

**Comment by rwedge** [comment]
I am willing to be a maintainer.

**Comment by jeff-hernandez** [comment]
I am willing to be a maintainer.

**Comment by davesque** [comment]
I am willing to be a maintainer.

**Comment by tamargrey** [comment]
@conda-forge/help-python This is ready for review!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**